src/Microsoft.Health.Fhir.Shared.Core/Features/Search/Expressions/Parsers/SearchParameterExpressionParser.cs

```diff
@@ -93,7 +93,7 @@ public class SearchParameterExpressionParser : ISearchParameterExpressionParser
                         string.Format(CultureInfo.InvariantCulture, Core.Resources.ModifierNotSupported, modifier, searchParameter.Name));
                 }
 
-                outputExpression = Expression.Contains(FieldName.TokenText, null, value, true);
+                outputExpression = Expression.StartsWith(FieldName.TokenText, null, value, true);
```

Comment:
Why was this changed to StartsWith? The spec says it should be partial search so shouldn't that be contains?

> For token: :text (the match does a partial searches on the text portion of a CodeableConcept or the display portion of a Coding), instead of the default search which uses codes. Other defined modifiers are :in, :below, :above and :not-in which are described below.

Reply:
See earlier thread:  https://github.com/microsoft/fhir-server/pull/520#discussion_r290797558